plugins/openvr/src/ViveControllerManager.cpp

```diff
@@ -2068,6 +2068,12 @@ controller::Input::NamedVector ViveControllerManager::InputDevice::getAvailableI
 }
 
 QString ViveControllerManager::InputDevice::getDefaultMappingConfig() const {
-    static const QString MAPPING_JSON = PathUtils::resourcesPath() + "/controllers/vive.json";
+    QString name(getOpenVrDeviceName().c_str());
+    QString MAPPING_JSON;
+    if (name.contains(QString("Vive")) || name.contains(QString("Vive"))) {
```

Comment:
```suggestion
    if (name.contains(QString("Vive")) {
```
This seems redundant.

Reply:
Thank you! Second "Vive" was intended to be "HTC" and would cause a bug otherwise.

Reply:
I fixed it in [dfef968](https://github.com/overte-org/overte/pull/481/commits/dfef9680f97cce6b65592fa6d5a97f251a17d53a)